src/themes/Mikings/ThankYouBox/ThankYouBox.scss

```diff
@@ -0,0 +1,17 @@
+@use 'src/components/ThankYouBox/ThankYouBox';
+
+.wmde-banner {
+	@include ThankYouBox.layout;
+
+	.thank-you-container {
+		font-weight: bold;
+		background: #ebe2fc;
```

Comment:
This is another thing we should create color variables for, and a flag I think.

But it also has me thinking, what do we do if they want to test 2 different colours of the box? We should chat about it in the daily tomorrow.